### PR TITLE
[WIP] Add buffer-product package for Cross-selling in the AppShell

### PR DESCRIPTION
**Pull request by CoxyBoris**
## Description

Heya Publish team 👋 
In order to do some cross-selling with the AppShell component, all products frontend need to have access to the global account from Core (which is only available from backends within the K8s cluster).

I created a small npm package to import on all products backend. This is temporary until Core provides a public api, but needed now by the AppShell component.

I'm not really sure how to test this PR and  I'm not used at all to your workflow in describing PRs or features and about the checklist here so feel to change /edit anything and your help will be very appreciated! 😅

Also I pinged the suggested reviewer here @josem but if someone else feels better I can change that! 😀

## Context & Notes

YOu can find more context on Cross selling in the AppShell in this [thread](https://threads.com/34368456655) 

## Checklist

-   [ ] My code follows the code style and guidelines of this project. 
-   [ ] I have added tests to cover my changes.
-   [ ] All new and existing tests passed.
-   [ ] I have tested different user stories. 
-   [ ] I have identified similar or related features and tested they are still working.
-   [ ] I have performed a self-review of my own code.
-   [ ] I have tested my changes/additions in the latest Chrome, Firefox, and Safari.
-   [ ] I have commented my code, particularly in hard-to-understand areas.

#### Staging Deployment

To visit the URL of the staging deployment, please click "Show all checks" at the bottom of this PR and click "details" next to `bufferbotbrains/cicd-buffer-publish-legacy`

### CODE REVIEW

**Comment by CoxyBoris** [comment]
Hey @josem Curious when you'll have a chance to take a look at this one? Also if I should assign another reviewer would you be up to let me know who would be the best person? 
Thanks a lot!
Pinging @jdeitchman here for visibility as well 😀

File: helm/buffer-publish.dev.yaml [hunk 1]
```diff
@@ -48,3 +48,8 @@ env:
         name: buffer-publish-keys
   - name: RELEASE_STAGE
     value: staging
+  # Auth Service env vars needed for buffer-products package
+  - name: AUTH_SERVICE_HOST
```
Comment (karelm) [inline helm/buffer-publish.dev.yaml]: Hey @CoxyBoris, is this value available available already? or do we need to expose it separately for this work?

**Comment by karelm** [comment]
Hey @CoxyBoris, just one small question for you around the environment variable, but I think its already there. Just checking really, otherwise LGTM!!! 👍

**Comment by hamstu** [comment]
@CoxyBoris @karelm is this still needing to be merged? 🤔

**Comment by CoxyBoris** [comment]
Hey @hamstu ! Great one , I think we can close this PR as it's not needed anymore, it was for Reply :)

**Pull request closed without merging**